sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageImplBuilder.java

```diff
@@ -39,93 +32,102 @@
 
 /** A builder for creating a new instance of the AzureBlobStorage type. */
 @ServiceClientBuilder(serviceClients = {AzureBlobStorageImpl.class})
-public final class AzureBlobStorageImplBuilder
-        implements HttpTrait<AzureBlobStorageImplBuilder>, ConfigurationTrait<AzureBlobStorageImplBuilder> {
+public final class AzureBlobStorageImplBuilder {
```

Comment:
Appears the version of AutoRest used to generate this was out-of-date, you'll want to update it and regenerate so the new features aren't lost